packages/panels/resources/views/components/layout/simple.blade.php

```diff
@@ -18,7 +18,7 @@ class="absolute end-0 top-0 flex h-16 items-center gap-x-4 pe-4 md:pe-6 lg:pe-8"
             </div>
         @endif
 
-        <div class="fi-simple-main-ctn w-full">
+        <div class="fi-simple-main-ctn w-full flex items-center flex-grow">
             <main
                 class="fi-simple-main mx-auto my-16 w-full bg-white px-6 py-12 shadow-sm ring-1 ring-gray-950/5 dark:bg-gray-900 dark:ring-white/10 sm:max-w-lg sm:rounded-xl sm:px-12"
```

Comment:
Could you try removing `mx-auto` and test if everything still works as expected? I think that class is no longer needed now we use flex for the parent, @atmonshi.

Reply:
then I have to add `justify-center` to the parent :) to center it 

<img width="986" alt="Screenshot 2023-08-04 at 4 27 03 PM" src="https://github.com/filamentphp/filament/assets/1952412/d0809542-1fdf-4668-94f1-ac23df358cb9">

Reply:
Oh I see, well both work.